ext/reflection/tests/gh8932.phpt

```diff
@@ -0,0 +1,24 @@
+--TEST--
+GH-8932 (Wrong closure scope class reported for static methods)
+--FILE--
+<?php
+class A {
+    public static function b() {
+        echo static::class;
+    }
+}
+
+class B extends A {}
+
+$c = ['B', 'b'];
+$d = \Closure::fromCallable($c);
+$r = new \ReflectionFunction($d);
+var_dump($r->getClosureScopeClass());
```

Comment:
`ReflectionMethod` returns `null`

https://3v4l.org/hXBYu

Reply:
I think this is to be expected; since there is no closure, there is no closure scope class. But in any way, [`ReflectionFunctionAbstract::getClosureScopeClass()`](https://www.php.net/manual/en/reflectionfunctionabstract.getclosurescopeclass.php) needs proper documentation.